Security.Platform.iOS/AgsSec.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using AeroGear.Mobile.Core.Utils;
+using AeroGear.Mobile.Security;
+
+namespace Security.Platform
+{
+    public class AgsSec
```

Comment:
Not in love with the class name.  I'm guessing it is like the other service classes where we will have an init method that takes in configuraiton parameters?  Actually I'm not sure what this class does. (Could be an effect of the WIP nature of this PR)